css-images-4/Overview.bs

```diff
@@ -1701,7 +1701,17 @@ Conic Gradient Examples</h4>
 			width: 200px; height: 200px;
 		</pre>
 
-		<img src="images/conic5.png" alt="" >
+		You can also achieve the same effect using "longer" hue interpolation. This is more concise and makes it easy to switch to other color spaces.
+
+		<pre class=lang-css>
+			background: radial-gradient(closest-side, gray, transparent),
+			            conic-gradient(in hsl longer hue, red 0 100%);
+			transform: scaleX(-1);
+			border-radius: 50%;
+			width: 200px; height: 200px;
+		</pre>
+
+		<img src="images/conic5.png" alt="color wheel">
```

Comment:
There are some differences between the current image (left) and the CSS rendering, should we update the image?

![image](https://github.com/user-attachments/assets/75afe023-149f-4e41-9e54-10728655e17a). <img width="200" src="https://github.com/user-attachments/assets/667005bb-1a07-47c1-ad8d-0ff4b2fd40fa">

Reply:
Yes, we should.

The rendering in Firefox (on Win, at least) also shows a rendering bug, but that is a separate issue

Reply:
@svgeesus You wrote:

> The rendering in Firefox (on Win, at least) also shows a rendering bug, but that is a separate issue

It looks OK to me on my Windows machine. Could you clarify what rendering bug you're seeing, please? (Or file a bug at https://bugzilla.mozilla.org with details.) Thanks!

Reply:
> Yes, we should.
@svgeesus Okay, I'll modify it.

Reply:
> It looks OK to me on my Windows machine. Could you clarify what rendering bug you're seeing, please?

![gradient-bug](https://github.com/user-attachments/assets/b9fe571a-e1f9-4739-b663-35d6aff2efba)

https://codepen.io/svgeesus/pen/KwKzgQE

Nightly 137.0a1 (2025-02-26) (64-bit) Windows 11